README.rst

```diff
@@ -0,0 +1,184 @@
+|Build Status| |Documentation Status| |codecov|
```

Comment:
Can we do this? https://coderwall.com/p/qawuyq/use-markdown-readme-s-in-python-modules